jekyll/_data/sidenav.yml

```diff
@@ -216,6 +214,8 @@ en:
       children:
         - name: Status Overview
           link: 2.0/status/
+        - name: Pipelines Overview
```

Comment:
I moved this page from the Projects section to the Pipelines section - seems a better fit?

Reply:
Definitely.